recipes/pandoc-plot/bld.bat

```diff
@@ -0,0 +1,2 @@
+if not exist %LIBRARY_BIN% mkdir %LIBRARY_BIN% || exit 1
+move pandoc-plot.exe %LIBRARY_BIN% || exit 1
```

Comment:
You need to build the binary, not repackage it.

Reply:
pandoc-plot is a Haskell binary. Building it from scratch would take much effort and time.

I'm following other recipes for Haskell binaries ([pandoc](https://github.com/conda-forge/pandoc-feedstock), [pandoc-crossref](https://github.com/conda-forge/pandoc-crossref-feedstock)) which make use of binaries from Github releases.

Reply:
Ok. Why only windows?

Reply:
That's because I haven't been able to generate MacOS and Linux binaries yet. I'm using Windows, so other binaries will have to be made via CI.